server/src/main/java/org/opensearch/search/profile/query/ProfileCollector.java

```diff
@@ -67,6 +69,8 @@ public ScoreMode scoreMode() {
         try {
             return super.scoreMode();
         } finally {
+            sliceStartTime = isStarted ? sliceStartTime : start;
+            isStarted = true;
```

Comment:
Nitpick, but I think the following logic is easier to follow and avoids doing assignments when isStarted is true. What do you think?

```suggestion
            if (isStarted == false) {
                sliceStartTime = start;
                isStarted = true;
            }
```

Reply:
Certainly, thank you, @andrross. I have now addressed the issue.